### PR TITLE
minor design changes on dropdown and OptionGroup

### PR DESCRIPTION
**Pull request by ChaitanyaGadodia**

### CODE REVIEW

File: src/components/styles/OptionGroup.styles.ts [hunk 1]
```diff
@@ -10,7 +10,13 @@ export const optionsWrapper = css({
...
   minWidth: 200,
   overflowY: "auto",
-  padding: "20px 1px"
+  padding: "20px 1px",
+  transition: "all 100ms linear",
+  willChange: "max-height"
+});
+
+export const optionsWrapperScrolledStyle = css({
+  maxHeight: 372
```
Comment (ritz078) [inline src/components/styles/OptionGroup.styles.ts]: this is a hack. not a solution.

File: src/components/styles/OptionGroup.styles.ts [hunk 2]
```diff
@@ -10,7 +10,15 @@ export const optionsWrapper = css({
   maxHeight: 352,
   minWidth: 200,
   overflowY: "auto",
-  padding: "20px 1px"
+  padding: "20px 1px",
+  transition: "all 100ms linear",
+  willChange: "max-height"
+});
+
+export const fixedDiv = css({
```
Comment (ritz078) [inline src/components/styles/OptionGroup.styles.ts]: a fixed div having position relative is ironical.

File: src/components/shared/OptionGroup.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ import {
   searchBoxScrolledStyle,
   searchBoxWrapper,
   optionsWrapper,
-  fixedDiv
+  fixedHeightDiv
```
Comment (ritz078) [inline src/components/shared/OptionGroup.tsx]: fixedHeightDiv, relativePositionedDiv, divHavingZIndex ?

why can this be searchBoxWrapper ?

Comment (ChaitanyaGadodia) [reply src/components/shared/OptionGroup.tsx]: Removed altogether.

**Comment by ritz078** [comment]
@ChaitanyaGadodia still do not merge ?

File: src/components/OptionGroupCheckBox.tsx [hunk 4]
```diff
@@ -1,37 +1,39 @@
...
-import * as mixins from "../theme/mixins";
 import Button from "./Button";
 import OptionGroup from "./shared/OptionGroup";
 import { OptionGroupCheckBoxProps } from "./typings/OptionGroupCheckBox";
+import * as styles from "../components/styles/OptionGroupCheckBox.styles";
 
 const OptionGroupCheckBox: React.SFC<OptionGroupCheckBoxProps> = props => {
   let { onApply, onClear, selected, onChange, ...rest } = props;
   return (
     <React.Fragment>
```
Comment (azizhk) [inline src/components/OptionGroupCheckBox.tsx]: Can remove this now

File: src/components/styles/OptionGroupCheckBox.styles.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import * as mixins from "../../theme/mixins";
+import { css } from "emotion";
+
+export const optionGroupCheckBoxWrap = css({
+  maxHeight: "532px",
```
Comment (azizhk) [inline src/components/styles/OptionGroupCheckBox.styles.ts]: Write code as to how this is computed.

**Pull request merged**